interface/patient_file/pos_checkout_normal.php

```diff
@@ -48,8 +48,8 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-require_once("../globals.php");
-require_once("$srcdir/patient.inc.php");
+require_once "../globals.php";
+require_once("$srcdir/patient.inc");
```

Comment:
revert above lines of code to original
(if the issue is that your current environment is missing the new *.inc.php files (ie. you are not working in most recent codebase), then you can get these scripts from master and put them in your current environment)